### PR TITLE
Create 0x6BFf2fE249601ed0Db3a87424a2E923118BB0312.json

### PR DESCRIPTION
**Pull request by beynbeyn**
Hi, I am new to GitHub but tried my best to follow the rules. Hope the formatting and the information provided is correct. If not please let me know! Thanks a lot for your time and effort!

### CODE REVIEW

File: 0x6BFf2fE249601ed0Db3a87424a2E923118BB0312.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "symbol": "FYZ",
+    "name": "Fyooz",
+    "address": "0x6BFf2fE249601ed0Db3a87424a2E923118BB0312",
+    "decimals": 18,
+    "website": "https://www.fyooz.io/",
+    "logo": "https://drive.google.com/file/d/1cqogQKwLoOjabwqcbKXDQR9kPlxpQ4ha/view?usp=sharing",
```
Comment (ligi) [inline 0x6BFf2fE249601ed0Db3a87424a2E923118BB0312.json:7]: this links to a website but not an image

File: 0x6BFf2fE249601ed0Db3a87424a2E923118BB0312.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "support": "hello@fyooz.io",
+    "social": {
+        "blog": "https://medium.com/fyooz",
+        "facebook": "https://www.facebook.com/fyooz.io/",
+        "instagram": "https://www.instagram.com/fyoozofficial",
+        "twitter": "https://twitter.com/fyooz_io",
```
Comment (ligi) [inline 0x6BFf2fE249601ed0Db3a87424a2E923118BB0312.json:13]: please remove the comma on the end

File: tokens/eth/0x6BFf2fE249601ed0Db3a87424a2E923118BB0312.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "symbol": "FYZ",
+    "name": "Fyooz",
+    "address": "0x6BFf2fE249601ed0Db3a87424a2E923118BB0312",
+    "decimals": 18,
+    "website": "https://www.fyooz.io/",
+    "logo": "https://fyooz-miscellaneous.s3.eu-central-1.amazonaws.com/logos/FYOOZ_Coin-Icon_White-on-Black_128x128.png",
+    "support": "hello@fyooz.io"
```
Comment (ligi) [inline tokens/eth/0x6BFf2fE249601ed0Db3a87424a2E923118BB0312.json:8]: there is a comma missing on this line

**Comment by ligi** [comment]
unfortunately I found one more error - the missing comma at support

**Comment by beynbeyn** [comment]
Added the comma

**Comment by beynbeyn** [comment]
Argh, its supposed to say "Add FYZ Token" ... sorry im such a noob.. this is my first time using github

File: tokens/eth/0x6BFf2fE249601ed0Db3a87424a2E923118BB0312.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+    "symbol": "FYZ",
+    "name": "Fyooz",
+    "address": "0x6BFf2fE249601ed0Db3a87424a2E923118BB0312",
+    "decimals": 18,
+    "website": "https://www.fyooz.io/",
+    "logo": "https://fyooz-miscellaneous.s3.eu-central-1.amazonaws.com/logos/FYOOZ_Coin-Icon_White-on-Black_128x128.png",
```
Comment (ligi) [inline tokens/eth/0x6BFf2fE249601ed0Db3a87424a2E923118BB0312.json:7]: sorry - did not see this when looking over it - but the CI found it - logo must be an object - see the other tokens for a reference

**Comment by beynbeyn** [comment]
I have made it as an object and also the support tab as an object aswell..
Sorry for the many mistakes i've made and thank you for your patience!

**Comment by ligi** [comment]
no worries - congratz for your first PR!

**Comment by beynbeyn** [comment]
Yay we've made it!! :) 
Do you know how long it will approximately take for MyCrypto and for Trezor to incorporate it?
Thanks for helping me out and have a nice day!

**Comment by ligi** [comment]
for TREZOR it should be in the next release - for myCrypto I do not know - maybe @blurpesec can give a hint here

**Pull request merged**